web/pdf_scripting_manager.js

```diff
@@ -96,17 +98,33 @@ class PDFScriptingManager {
     this._scripting = this._createScripting();
 
     this._internalEvents.set("updatefromsandbox", event => {
+      if (event?.source !== window) {
+        return;
+      }
```

Comment:
While this is probably not *necessary*, it a "better safe than sorry"-type of check I forgot to add in PR #13042